src/vs/workbench/electron-browser/commands.ts

```diff
@@ -423,3 +424,24 @@ export function registerCommands(): void {
 		});
 	});
 }
+
```

Comment:
I think you can remove the `namespace GlobalQuickOpenCommand` here, not sure why we would want it.